src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/RichTextBoxTests.cs

```diff
@@ -10646,6 +10646,33 @@ public void RichTextBox_OnGotFocus_RaisesAutomationNotification_WithText(EventAr
             Times.Once);
     }
 
+    [WinFormsTheory]
+    [InlineData(-1, -1, 0)]
+    [InlineData(0, 0, 0)]
+    [InlineData(10, 10, 1)]
+    public void RichTextBox_GetCharIndexFromPosition_Invoke_ReturnsExpected(int x, int y, int expectedIndex)
+    {
+        using RichTextBox richTextBox1 = new();
+        richTextBox1.Text = "Hello, World!";
+
+        Assert.NotEqual(IntPtr.Zero, richTextBox1.Handle);
```

Comment:
We can now treat window handles as `nint`s , since NET8 IntPtrs and nints are equivalent.
```suggestion
        Assert.NotEqual(0, richTextBox1.Handle);
```